scm/driver/github/webhook.go

```diff
@@ -352,6 +439,95 @@ type (
 		Repository repository `json:"repository"`
 		Sender     user       `json:"sender"`
 	}
+
+	pipelineHook struct {
+		Action      string `json:"action"`
+		WorkflowRun struct {
+			ID               int64   `json:"id"`
+			Name             string  `json:"name"`
+			NodeID           string  `json:"node_id"`
+			HeadBranch       string  `json:"head_branch"`
+			HeadSHA          string  `json:"head_sha"`
+			Path             string  `json:"path"`
+			DisplayTitle     string  `json:"display_title"`
+			RunNumber        int     `json:"run_number"`
+			Event            string  `json:"event"`
+			Status           string  `json:"status"`
+			Conclusion       *string `json:"conclusion"`
```

Comment:
remove pointer, use `null.String` instead (see examples in other files)

Reply:
Done